web/app/css/styles.css

```diff
@@ -11,6 +11,8 @@
     --latency-p95: #2D9CDB;
     --latency-p50: #56CCF2;
     --subheader-grey: #828282;
+    --status-gray: #BDBDBD;
+    --green: #27AE60;
```

Comment:
These colors are still referenced by their hex code in a few other places -- would it be worth switching those to use these variables?

```
$ grep -Erl '(27AE60|BDBDBD)' web/app/css
web/app/css/cta.css
web/app/css/health-pane.css
web/app/css/service-mesh.css
web/app/css/styles.css
```